linker_scripts/object_script.txt

```diff
@@ -54,6 +54,10 @@ D_06009120 = 0x06009120;
 D_06005458 = 0x06005458;
 D_0600788C = 0x0600788C;
 
+/* z_en_tubo_trap */
+D_05018090 = 0x05018090;
+D_05017EA0 = 0x05017EA0;
+
```

Comment:
These are in segment 5, so they're not actually in the `en_tubo_trap` actor. Move them to the top of the file, under the `0x04` addresses.
It's actually unclear what object they're from, since segment 5 could be 1 of 2. But since these are enemies, I would guess it's `gameplay_dangeon_keep`, so use that as the comment for what these are from.